.github/workflows/chart-automation-tests.yaml

```diff
@@ -0,0 +1,31 @@
+name: charts-automation tests
+
+on:
+  pull_request:
+    branches:
+      - master
+    paths:
+      - 'charts/sysdig-deploy/**'
+      - 'charts/admission-controller/**'
+      - 'charts/node-analyzer/**'
+      - 'charts/kspm-collector/**'
+      - 'charts/rapid-response/**'
+
+jobs:
+  helm-charts-validation-test:
+    runs-on: tools-runner
```

Comment:
Is this a custom runner that needs to be created before this is merged?

Reply:
you may be right...